### PR TITLE
fix #515 move v-on listeners from router-link to child div

### LINKED ISSUE #515: va-button doesn't use vue-router history mode as expected
* **What is the expected behavior?**
after clicking on the button, the location should be changed without reloading

* **What is the current behavior?**
after clicking on the button, the location changes with reloading

* **Steps to reproduce, ideally also a screenshot or gif if it's a style issue**
[codesandbox](https://codesandbox.io/s/qye05?file=/src/App.vue)
[website](https://vuestic-test.vercel.app)
play with buttons:
image

* **Other information**

### PR DESCRIPTION
**Pull request by m0ksem**
## Description
closes #515 

`v-on="inputListeners"` was bind to route-link (`<component  tag="router-link" />`).

Route-link does not support `v-on`, therefore we need to use `.native` modifier (for vue2)  or move all listeners to a div wrapper. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/vuestic-components/va-chip/VaChip.vue [hunk 1]
```diff
@@ -1,40 +1,44 @@
...
-    :to="to"
-    :replace="replace"
-    :exact="exact"
-    :active-class="activeClass"
-    :exact-active-class="exactActiveClass"
-    class="va-chip"
-    :class="computedClass"
-    :style="computedStyle"
-    :tabindex="indexComputed"
+  <div
```
Comment (eqxs) [inline packages/ui/src/components/vuestic-components/va-chip/VaChip.vue:2]: i think this element should be 'inline-flex', like a child element

**Review by eqxs — CHANGES_REQUESTED** [review]
please check the comment

File: packages/ui/src/components/vuestic-components/va-chip/VaChip.vue (hunk 1 above)
Comment (m0ksem) [reply packages/ui/src/components/vuestic-components/va-chip/VaChip.vue]: fixed in https://github.com/epicmaxco/vuestic-ui/pull/516/commits/d1245b9d935941edb873b3ba355ba74d65006006

**Review by eqxs — APPROVED** [review]
good

**Pull request merged**